### DISCUSSION TITLE
Add Support for Baidu Wenxin Qianfan's Embedding and Rerank Models to Dify

### DISCUSSION THREAD

**Discussion by ninehills** [langgenius/dify#3341] [Suggestion]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] Pleas do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing?

Due to the need to use specific model providers in our company, I would like Dify to support Baidu Wenxin Qianfan's embedding and rerank models.

### 2. Describe the feature you'd like to see

I would like Dify to add support for Baidu Wenxin Qianfan's embedding and rerank models, similar to the existing support for models like Cohere. This would allow us to use these Baidu models on the Dify platform.

### 3. How will this feature improve your workflow or experience?

This feature would allow us to better integrate Dify into our company's workflow, thereby improving work efficiency.

### 4. Additional context or comments

None.

### 5. Can you help us with this feature?

- [X] I am interested in contributing to this feature.

**Comment by LVHAI** [comment]
I also need Dify to support Baidu Wenxin Qianfan's Embedding and Rerank Models

**Comment by BradKML** [comment]
How about focus on getting support for multiple rerankers instead of only a single model?
